src/CssInliner.php

```diff
@@ -690,7 +691,16 @@ private function parseCssRules(string $css): array
             }
         }
 
-        \usort($cssRules['inlinable'], [$this, 'sortBySelectorPrecedence']);
+        \usort(
+            $cssRules['inlinable'],
+            /**
+             * @param array<string, string> $a
+             * @param array<string, string> $b
+             */
```

Comment:
I think the existing DocBlock type hint (`string[]`) for `sortBySelectorPrecedence` parameters was incorrect.  In full they should be `array{media: string, selector: string, hasUnmatchablePseudo: bool, declarationsBlock: string, line: int}` but probably only the keys used by the function need to be specified, i.e. `array{selector: string, line: int}`.

Reply:
Fixed and repushed (both instances).